test/y2storage/initial_guided_proposal_test.rb

```diff
@@ -193,6 +194,353 @@
         end
       end
 
+      context "and the proposal is set to use a multidisk_first approach" do
```

Comment:
Thought: maybe it's time to split this long test file for the sake of legibility. Having this kind of "special" scenarios in a separated file looks good to me.